README.rst

```diff
@@ -2116,6 +2116,101 @@ Remember to change ``geo_views`` location in ``urls.py`` in point 11 for extendi
 
 For more information about django views, please refer to the `views section in the django documentation <https://docs.djangoproject.com/en/dev/topics/http/views/>`_.
 
+Registering / Unregistering Commands
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+OpenWISP Controller provides registering and unregistering commands
+through utility functions ``openwisp_controller.connection.commands.register_command``
+and ``openwisp_notifications.types.unregister_notification_type``.
+Using these functions you can register or unregister commands from your code.
```

Comment:
Do you remember we are trying to help out non programmers to understand what OpenWISP does and how to customize it, right?

Therefore, we should try to go straight to the point.

I'd write something like:

> If you need to allow your users to quickly send specific commands that are used often in your network regardless of your users' knowledge of Linux shell commands, you can register new custom commands or unregister commands you don't need with the functions described in this section.

Maybe we should put the example and screenshot here?

I would also try recommending where they can put these instructions, does putting this in `settings.py` work?
If yes, we can recommend that option which is the easiest.

Reply:
> Do you remember we are trying to help out non programmers to understand what OpenWISP does and how to customize it, right?

Ok, but this seems more like a programming feature. I don't think one can register commands from `settings.py`. They'll have to put this code in some ready function. 

My goal here was to write a documentation and not a tutorial, because I didn't expect a user of OpenWISP (developer deploying OpenWISP) adding new commands. 

We can also add a dedicated setting for specifying new commands. That will be most user friendly and won't require much programming knowledge on Python part but will still require knowledge of JSONSchema. 

But, I also believe that if we can make it easy we should make it easy.

Reply:
I checked and calling reigster/unregister command functions works in settings.py. But, should we encourage users to do that? I don't think so.